core/src/main/java/jenkins/slaves/DefaultJnlpSlaveReceiver.java

```diff
@@ -216,5 +216,5 @@ public void setLog(@Nonnull OutputStream log) {
 
     private static final Logger LOGGER = Logger.getLogger(DefaultJnlpSlaveReceiver.class.getName());
 
-    private static final String COOKIE_NAME = JnlpSlaveAgentProtocol2.class.getName()+".cookie";
```

Comment:
Given the old cookie name, is this only used for JNLP Agent Protocol/2? Or was it just introduced there, and is still used in later protocols?

Reply:
It was introduced with an earlier protocol and use was continued, at least somewhat, with newer ones. Because of the way the cookie is roundtripped, the exact name doesn't matter so I cleaned it up a bit.